lib/streamlit/runtime/memory_media_file_storage.py

```diff
@@ -17,10 +17,15 @@
 import contextlib
```

Comment:
This file was moved from `streamlit.web.server` to `streamlit.runtime`, because it's not actually web-specific.